packages/components/src/Tabs/Tabs.tsx

```diff
@@ -38,21 +33,42 @@ export const Tabs = ({
     size,
     variant,
   });
-
   return (
     <TabContext.Provider value={{ classNames }}>
-      <div
-        className={cn('flex', gapSpace[space], classNames.tabs)}
-        {...tabListProps}
-        ref={ref}
-      >
-        {[...state.collection].map(item => {
-          return <Tab key={item.key} item={item} state={state} />;
-        })}
+      {/* tabs container */}
+      <div className={className}>
+        <div
+          className={cn('flex', gapSpace[space], classNames.tabs)}
+          {...tabListProps}
+          ref={ref}
+        >
+          {[...state.collection].map(item => {
+            return <Tab key={item.key} item={item} state={state} />;
+          })}
+        </div>
+        <TabPanel
+          key={state.selectedItem?.key}
+          state={state}
+          className={state.selectedItem?.props?.className}
```

Comment:
Can we do this via an aria selector instead?

Reply:
@sebald 
can't use it 
i moved it to `TabPanel` too , since we don't need to pass it down to it